pipelined/bevy_ui2/Cargo.toml

```diff
@@ -10,23 +10,27 @@ keywords = ["bevy"]
 
 [dependencies]
 # bevy
-bevy_app = { path = "../bevy_app", version = "0.5.0" }
-bevy_asset = { path = "../bevy_asset", version = "0.5.0" }
-bevy_core = { path = "../bevy_core", version = "0.5.0" }
-bevy_derive = { path = "../bevy_derive", version = "0.5.0" }
-bevy_ecs = { path = "../bevy_ecs", version = "0.5.0" }
-bevy_input = { path = "../bevy_input", version = "0.5.0" }
-bevy_log = { path = "../bevy_log", version = "0.5.0" }
-bevy_math = { path = "../bevy_math", version = "0.5.0" }
-bevy_reflect = { path = "../bevy_reflect", version = "0.5.0", features = ["bevy"] }
-bevy_render = { path = "../bevy_render", version = "0.5.0" }
-bevy_sprite = { path = "../bevy_sprite", version = "0.5.0" }
-bevy_text = { path = "../bevy_text", version = "0.5.0" }
-bevy_transform = { path = "../bevy_transform", version = "0.5.0" }
-bevy_window = { path = "../bevy_window", version = "0.5.0" }
-bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
+bevy_app = { path = "../../crates/bevy_app", version = "0.5.0" }
+bevy_asset = { path = "../../crates/bevy_asset", version = "0.5.0" }
+bevy_core = { path = "../../crates/bevy_core", version = "0.5.0" }
+bevy_core_pipeline = { path = "../bevy_core_pipeline", version = "0.5.0" }
+bevy_derive = { path = "../../crates/bevy_derive", version = "0.5.0" }
+bevy_ecs = { path = "../../crates/bevy_ecs", version = "0.5.0" }
+bevy_input = { path = "../../crates/bevy_input", version = "0.5.0" }
+bevy_log = { path = "../../crates/bevy_log", version = "0.5.0" }
+bevy_math = { path = "../../crates/bevy_math", version = "0.5.0" }
+bevy_reflect = { path = "../../crates/bevy_reflect", version = "0.5.0", features = ["bevy"] }
+bevy_render2 = { path = "../bevy_render2", version = "0.5.0" }
+bevy_sprite2 = { path = "../bevy_sprite2", version = "0.5.0" }
+bevy_text2 = { path = "../bevy_text2", version = "0.5.0" }
+bevy_transform = { path = "../../crates/bevy_transform", version = "0.5.0" }
+bevy_window = { path = "../../crates/bevy_window", version = "0.5.0" }
+bevy_utils = { path = "../../crates/bevy_utils", version = "0.5.0" }
 
 # other
 stretch = "0.3.2"
 serde = {version = "1", features = ["derive"]}
 smallvec = { version = "1.6", features = ["union", "const_generics"] }
+bytemuck = { version = "1.5", features = ["derive"] }
+wgpu = "0.11.0"
```

Comment:
New render features generally shouldn't need to add a direct wgpu dependency. If we're missing wgpu exports, can you add them to the existing re-exports in bevy_render2::render_resources?

Reply:
Done in 8c19df4, it seems everything was exported correctly (some might have been missing early on).